docs/content_management/pages/page_builder_guide.md

```diff
@@ -43,6 +43,8 @@ It's also important to choose a layout - it determines the arrangement of drop z
 
 ![Page Builder - diagram](page_builder_diagram.png)
 
+<!--ARCADE EMBED START--><div style="position: relative; padding-bottom: calc(51.27314814814815% + 41px); height: 0; width: 100%;"><iframe src="https://demo.arcade.software/kbdnGkdrkXL2VAJW6c3O?embed&embed_mobile=tab&embed_desktop=inline&show_copy_link=true" title="Page Builder interface" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen allow="clipboard-write" style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; color-scheme: light;" ></iframe></div><!--ARCADE EMBED END-->
```

Comment:
Since the name of this animation if Page Builder interface, I would move it to the "Page Builder interface" section (line 54), for example, next to the form_builder_interface.png file.